src/test/java/org/apache/drill_web_test_framework/rest_api/tests/secure/ProfilesTest.java

```diff
@@ -89,14 +99,14 @@ public void nonAdminCancelSecondNonAdminQuery() throws InterruptedException {
     //Cancel the query with the queryId
     given()
         .pathParam("queryID", queryId)
-        .filter(nonAdminSessionFilter)
+        .filter(secondNonAdminSessionFilter)
         .when()
         .get("/profiles/cancel/{queryID}")
         .then()
-        .statusCode(200)
-        .body(containsString("Failure attempting to cancel query" + " " + queryId));
-    //put 5 seconds sleep time
-    Thread.sleep(5000);
+        .statusCode(500)
+        .body(containsString("PERMISSION ERROR: Not authorized to cancel the query '" + queryId + "'"));
+    //put 3 seconds sleep time
+    Thread.sleep(3000);
```

Comment:
Thread.sleep is forbidden. Try to use something else :)
Or exclude this test until you figured out how to make it right.

Reply:
Done.